en/test_and_ci/unit_tests.md

```diff
@@ -42,7 +42,7 @@ The steps to creating new functional tests are as follows:
 1. If additional library dependencies are required, they should also be added to the CMakeLists after the `LINKLIBS` as shown above.
 
 Tests can be run via `make tests`, after which you will find the binary in `build/px4_sitl_test/functional-MyNewFunctional`.
-It can be run directly in a debugger, however be careful to only run one test per executable invocation using the [--gtest_filter=\<regex\>](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#running-a-subset-of-the-tests) arguments, as some parts of the uORB and parameter libraries don't clean themselves up perfectly and may result in undefined behavior if set up multiple times.
+It can be run directly in a debugger, however be careful to only run one test per executable invocation using the [--gtest_filter=\<regex\>](https://github.com/google/googletest/blob/master/docs/advanced.md#running-a-subset-of-the-tests) arguments, as some parts of the uORB and parameter libraries don't clean themselves up perfectly and may result in undefined behavior if set up multiple times.
```

Comment:
@MaEtUgR Roman wrote on Slack:

I don't understand this part `--gtest_filter=\<regex\>`
Does this work?
For me only this works: `make tests TESTFILTER=functional-Takeoff`

Reply:
Note, Ah, I see further answer.